core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -165,14 +165,14 @@ sealed abstract class IO[+E, +A] { self =>
    * then the action will fail with some error.
    */
   final def race[E1 >: E, A1 >: A](that: IO[E1, A1]): IO[E1, A1] =
-    raceBoth(that).map(_.merge)
+    raceEither(that).map(_.merge)
 
   /**
    * Races this action with the specified action, returning the first
    * result to produce a value, whichever it is. If neither action succeeds,
    * then the action will fail with some error.
    */
-  final def raceBoth[E1 >: E, B](that: IO[E1, B]): IO[E1, Either[A, B]] =
+  final def raceEither[E1 >: E, B](that: IO[E1, B]): IO[E1, Either[A, B]] =
```

Comment:
my 2 cents: there can only be one winner in a race, so I think `Either` is part of the definition. Hence I would call this one `race` and the other `raceMerge` or some such.